Patch-Source: https://github.com/cntools/libsurvive/pull/292.patch
--
From c955949969a66ffe78215385ea91374cd87b7093 Mon Sep 17 00:00:00 2001
From: Simon Zeni <simon.zeni@collabora.com>
Date: Wed, 4 Oct 2023 14:13:10 -0400
Subject: [PATCH] src/driver_usbmon: replace off64_t by z_off_t

`off64_t` is not a language defined type and is not available everywhere. Instead
the function should use `z_off_t` which is defined by zlib (and is most likely
`off_t`)
---
 src/driver_usbmon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/driver_usbmon.c b/src/driver_usbmon.c
index 22102c44..a476afe7 100644
--- a/src/driver_usbmon.c
+++ b/src/driver_usbmon.c
@@ -790,7 +790,7 @@ static int gzip_cookie_close(void *cookie) { return gzclose((gzFile)cookie); }
 
 static ssize_t gzip_cookie_read(void *cookie, char *buf, size_t nbytes) { return gzread((gzFile)cookie, buf, nbytes); }
 
-int gzip_cookie_seek(void *cookie, off64_t *pos, int __w) { return gzseek((gzFile)cookie, *pos, __w); }
+int gzip_cookie_seek(void *cookie, z_off_t *pos, int __w) { return gzseek((gzFile)cookie, *pos, __w); }
 
 cookie_io_functions_t gzip_cookie = {
 	.close = gzip_cookie_close, .write = gzip_cookie_write, .read = gzip_cookie_read, .seek = gzip_cookie_seek};
